src/develop/ui/patterns/web/utilities/animate.md

```diff
@@ -5,89 +5,42 @@ summary: Animate adds default animations to emphasize elements as they appear on
 
 # Animate
 
-Create animations on elements inside the placeholder.
+You can use the Animate UI Pattern to create animations within your app. This UI pattern allows you to emphasize elements as they appear on screen which enhances the overall usability of the app. 
 
-Use Animate to add default animations to emphasize elements as they appear on the screen. Animations should resemble familiar real-life movements, helping the user understand the interface.  
+![](<images/animate-image-10.png>)
 
-**How to use**
+**How to use the Animate UI Pattern**
 
-Drag the content inside the placeholder and configure the animation type of the Block. You can also define the start time and duration of the animation.
+1. In Service Studio, in the Toolbox, search for `Animate`. 
 
-1. Drag the Animate pattern into the preview.
+    The Animate widget is displayed.
 
-1. Set the content you need on the placeholder.
+    ![](<images/animate-image-11.png>)
 
-    ![](<images/animate-image-1.png>)
+1. From the Toolbox, drag the Animate widget into the Main Content area of your application's screen.
+    
+    [](<images/animate-image-1.png>)
 
-1. Set the Input Parameters to extend the default values.
+1. On the **Properties** tab, set the relevant properties, for example, where you want the animation to enter and leave the screen.
 
     ![](<images/animate-image-2.png>)
 
```

Comment:
We should add here an example with some content. An image or something like that :)